cmd/config.go

```diff
@@ -303,6 +303,47 @@ func getServiceConfig(reader *bufio.Reader) (*v3.RKEConfigServices, error) {
 		return nil, err
 	}
 	servicesConfig.Kubelet.InfraContainerImage = infraPodImage
+
+	// Add the flexvolume mount to kublet ExtraBinds
+	flexVolMount, err := getConfig(reader, "Enable the FlexVolume driver mount", "no")
+	if err != nil {
+		return nil, err
+	}
+
+	if strings.ContainsAny(flexVolMount, "Yes y Y yes") {
+		gkeBind := "/home/kubernetes/flexvolume:/home/kubernetes/flexvolume"
+		rkeBind := "/var/lib/kubelet/volumeplugins:/var/lib/kubelet/volumeplugins"
```

Comment:
is this parameter something that can be configured on RKE? if yes, can we reference instead of hardcoding here?

Reply:
Found the reference here: https://github.com/rancher/rke/pull/474/files#diff-31f9f805fb3e45c114bdaa74f9447759R315. Perhaps we can create a global variable, and use it in all places

Reply:
I pulled these paths from the longhorn README. Looking back they definitely should not be hard coded. Reading the [Flexvolume Docs](https://github.com/kubernetes/community/blob/master/contributors/devel/flexvolume.md) should I add `--volume-plugin-dir` to the extra args for the kubelet as well when the user wants to enable Flexvolume mount. I know I have seen some issues in slack asking how to set these values with rke as well. Thoughts? @alena1108

Reply:
@dahendel `should I add --volume-plugin-dir to the extra args for the kubelet as well when the user wants to enable Flexvolume mount` - yes. This parameter as an extra arg would override https://github.com/rancher/rke/blob/9e32c532e8c426dbb8281da8389ca12a9ceb1e0b/cluster/plan.go#L308